### PR TITLE
Setup MongoDB, create basic user model and update Readme

### PR DESCRIPTION
**Pull request by ValEmpire**

### CODE REVIEW

File: server/models/user.model.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+		type: String,
+		required: true,
+	},
+	role: {
+		type: String,
+		required: true,
+	},
+});
+
+const User = mongoose.model("user", userSchema);
```
Comment (dproc96) [inline server/models/user.model.js]: For Mongo the convention is generally to capitalize model names, like "User"

File: server/app.js [hunk 2]
```diff
@@ -21,19 +33,19 @@ app.use("/", indexRouter);
 app.use("/ping", pingRouter);
 
 // catch 404 and forward to error handler
-app.use(function(req, res, next) {
-  next(createError(404));
+app.use(function (req, res, next) {
+	next(createError(404));
```
Comment (dproc96) [inline server/app.js:37]: Can you check the spacing on your linter? It looks like it's set to 4 instead 2

**Review by dproc96 — APPROVED** [review]
Had two small comments, but pending those this is good to go

File: server/models/user.model.js (hunk 1 above)
Comment (ValEmpire) [reply server/models/user.model.js]: Got it. Thank you.

**Pull request merged**